### PR TITLE
Create changelog for v2.24.0 release

### PR DESCRIPTION
**Pull request by adambarreiro**
This PR prepares the changelog for the v2.24.0 release, which is focused mainly in bug fixes.

### CODE REVIEW

File: .changes/v2.24.0/657-features.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
-Added method `Client.QueryVappNetworks` to retrieve all vApp networks [GH-657]
-Added `VApp` methods `QueryAllVappNetworks`, `QueryVappNetworks`, `QueryVappOrgNetworks` to retrieve various types of vApp networks [GH-657]
+* Added method `Client.QueryVappNetworks` to retrieve all vApp networks [GH-657]
```
Comment (dataclouder) [inline .changes/v2.24.0/657-features.md:1]: How about we move this change item to the next release and leave only bug fixes in this one?

**Review by Didainius — APPROVED** [review]

**Review by lvirbalas — APPROVED** [review]

File: .changes/v2.24.0/657-features.md (hunk 1 above)
Comment (adambarreiro) [reply .changes/v2.24.0/657-features.md:1]: I think it should be good/useful to have them, what do others think?
@lvirbalas @Didainius

**Review by dataclouder — APPROVED** [review]

File: .changes/v2.24.0/657-features.md (hunk 1 above)
Comment (dataclouder) [reply .changes/v2.24.0/657-features.md:1]: I am OK with either choice

Comment (lvirbalas) [reply .changes/v2.24.0/657-features.md:1]: I'm OK with this, don't want to revert at this point.

**Pull request merged**